framework/src/modules/pos/module.ts

```diff
@@ -265,6 +265,7 @@ export class PoSModule extends BaseModule {
 			{},
 			{
 				...defaultConfig,
+				roundLength: defaultConfig.numberActiveValidators + defaultConfig.numberStandbyValidators,
```

Comment:
this will be wrong because in the `moduleConfig` active/standby validator might be changed, but round length will be based on the default value